### PR TITLE
FeatureForms: Adds TextFormElement support

### PR DESCRIPTION
**Pull request by dotMorten**
Note: This PR adds basic text support. Markdown is currently stripped.

### CODE REVIEW

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        {
+            return Regex.Replace(markdownText, @"(\*\*|__)(.*?)\1", "$2")
+                        .Replace("*", "")
+                        .Replace("_", "")
+                        .Replace("`", "")
+                        .Replace("~", "")
+                        .Replace("<br>", "\n")
+                        .Replace("<br/>", "\n")
+                        .Replace("\n", Environment.NewLine);
+        }
```
Comment (mstefarov) [inline src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.cs:133]: This is not great &mdash; you'd need a dozen carefully-ordered regexes to cover markdown syntax &mdash; but I understand that it's a placeholder for something better.

**Review by mstefarov — APPROVED** [review]

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.cs (hunk 1 above)
Comment (dotMorten) [reply src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.cs:133]: Yes I'm aware - it'll be gone before we ship.

**Review by prathameshnarkhede — APPROVED** [review]
Looks good

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.Maui.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            NameScope.SetNameScope(readonlyText, nameScope);
+            nameScope.RegisterName("ReadOnlyText", readonlyText);
+            return readonlyText;
+        }
+
+        /// <inheritdoc />
+        protected override void OnApplyTemplate()
+        {
+            base.OnApplyTemplate();
+            _readonlyLabel = GetTemplateChild("ReadOnlyText") as Label;
```
Comment (prathameshnarkhede) [inline src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.Maui.cs:35]: Just a suggestion. Can we move these Element Names to a constant? We use them in couple more places across platform.
e.g. `ReadOnlyText` to its base partial `TextFormElementView.cs`

Comment (dotMorten) [reply src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.Maui.cs:35]: Yes I believe it's already fixed in the next PR that adds markdown

**Pull request merged**